google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UpdateAudienceTargetRestriction.java

```diff
@@ -185,7 +213,9 @@ private void runExample(GoogleAdsClient googleAdsClient, long customerId, long a
               Long.toString(customerId), ImmutableList.of(operation));
       // Prints the resource name of the updated object.
       System.out.printf(
-          "Updated ad group with resourceName: '%s'.%n", response.getResults(0).getResourceName());
+          "Updated targeting setting of ad group with resourceName '%s'; set the AUDIENCE "
+              + "target restriction to bid_only.%n",
```

Comment:
"Set the AUDIENCE targeting restriction to "Observation""
Or
"Set the bid value field of AUDIENCE targeting restriction to true".

would be more accurate.